### PR TITLE
HYDRATOR-1498 Make transaction isolation level configurable

### PR DESCRIPTION
**Pull request by Denton-L**
JIRA: https://issues.cask.co/browse/HYDRATOR-1498
Bamboo: https://builds.cask.co/browse/HYP-BAD285

### CODE REVIEW

File: database-plugins/widgets/Database-batchsource.json [hunk 1]
```diff
@@ -52,6 +52,21 @@
...
             "default": "false"
           }
+        },
+        {
+          "widget-type": "select",
+          "label": "Database Isolation Level",
+          "name": "databaseIsolationLevel",
+          "widget-attributes": {
+            "values": [
+              "TRANSACTION_NONE",
```
Comment (nitinmotgi) [inline database-plugins/widgets/Database-batchsource.json]: Not user friendly !!!

**Review by nitinmotgi — CHANGES_REQUESTED** [review]
Can you please make it user friendly.

File: database-plugins/widgets/Database-batchsource.json (hunk 1 above)
Comment (Denton-L) [reply database-plugins/widgets/Database-batchsource.json]: What would a good alternative be?

Comment (Denton-L) [reply database-plugins/widgets/Database-batchsource.json]: It's all selected from a dropdown menu so the user shouldn't have to enter anything manually. Plus, the default is the expected value: `TRANSACTION_SERIALIZABLE`.

**Comment by gsps1** [comment]
will be good to update the docs about this property `Database-batchsource.md`

**Comment by Denton-L** [comment]
@nitinmotgi Updated it to make it more user-friendly.

**Comment by awholegunch** [comment]
Docs LGTM, but someone else should check my changes.

**Pull request closed without merging**